src/effects/VST3/internal/x11/RunLoop.cpp

```diff
@@ -0,0 +1,119 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file RunLoop.cpp
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#include "RunLoop.h"
+
+using namespace internal::x11;
+
+IMPLEMENT_FUNKNOWN_METHODS(RunLoop, Steinberg::Linux::IRunLoop, Steinberg::Linux::IRunLoop::iid);
+
+gboolean RunLoop::OnEvent(GIOChannel *source, GIOCondition condition, gpointer data)
```

Comment:
What is `condition` for ?  It is ignored

Reply:
It's not used by the event handler